Dockerfile

```
@@ -13,13 +12,13 @@ RUN mkdir src && \
 COPY build.rs ./
 COPY appinfo/info.xml ./appinfo/
 COPY src/ ./src/
-RUN sudo chown -R rust:rust . && touch src/main.rs
+RUN touch src/main.rs
 
 RUN cargo build --release
 
 FROM scratch
 
-COPY --from=build /home/rust/src/target/x86_64-unknown-linux-musl/release/notify_push /
+COPY --from=build /volume/target/x86_64-unknown-linux-musl/release/notify_push /
 EXPOSE 7867
 
 CMD ["/notify_push"]
```

```suggestion
CMD ["/notify_push"]

```

Please add a new line at the end of the file